### ISSUE TITLE
Team creation under Firefox

### ISSUE THREAD

**Issue by bslbckr** [ultical/ultical#6]
we got some reports that the creation of teams using KUbuntu and Firefox is not working. The same behavior could be observed for Firefox running under Android. 

I tried to re-produce the described behavior and failed. I used Firefox 43.0.4 running at an Ubuntu 15.10 

I think we need more information on this topic in order to re-produce it first and then finally fix it.

**Comment by bslbckr** [comment]
Is this issues solved with 6587b80d55577b8030e59a6a5ee9e1c235606cf4 @bastrapp ?

**Comment by bastrapp** [comment]
No, it addresses a different issue that I encoutered with Firefox in Ubuntu. The team-creation issue I coudn't reproduce.

Just a wild guess: Maybe it just so happened to be at a time when the connection pool was empty (see issue #7) and when the reporter tried it later with a different device, the server was restarted...
